types/mapbox-gl/index.d.ts

```diff
@@ -161,6 +161,8 @@ declare namespace mapboxgl {
         loaded(): boolean;
 
         remove(): void;
+		
+		triggerRepaint(): void;
```

Comment:
Please fix this line to use the same indentation style (spaces) as the rest of the file uses